ConsoleApp1/Program.cs

```diff
@@ -10,134 +10,192 @@ class Program
-                Console.ReadLine();
+                Console.WriteLine(">You see a firearm sitting next to you");
+                Console.WriteLine("(Press enter to continue...)");
+                Console.ReadKey();
 
-                Console.WriteLine(">You wake up in a the hospital bed once again, this time much sooner than before, a nurse is checking up on you and you try to get her attention<");
-                Console.WriteLine("(Press enter to continue)");
-                Console.ReadLine();
+                Console.WriteLine("Do you pick it up or leave it there?");
+                Console.WriteLine("(take/leave)");
+                string answer;
+                answer = Console.ReadLine();
 
-                Console.WriteLine(">The nurse went off to get the doctor, five minutes later he arrives<");
-                Console.WriteLine("(Press enter to continue)");
-                Console.ReadLine();
+                if (answer == "take")
+                {
+                    Console.WriteLine(">You take the firearm and rush to the door");
+                    Console.WriteLine("(Press enter to continue)");
+                    Console.ReadKey();
+
+                    Console.WriteLine(">As you enter the next room you see three men - all of them unfriendly");
+                    Console.WriteLine("What do you do? Shoot or don't shoot?");
+                    string answer2 = Console.ReadLine();
+                                        
+                    if (answer2 == "Shoot")
+                    {
+                        Console.WriteLine();
+                        Console.WriteLine(">You shot all three of them with the weapon you picked up, killing them - you can move on");
+                    }
+                    else if (answer2 == "Don't shoot")
+                    {
+                        Console.WriteLine();
+                        Console.WriteLine(">Your newfound friend shot them all, killing them - you can move on");
+                    }
+                    Console.WriteLine();
+                    Console.WriteLine("That was close.. they could have killed us both!");
 
 
-                Console.WriteLine(">The doctor walks in and looks at you, you sit up in the bed and watch what he's doing<");
-                Console.WriteLine("(Press enter to continue)");
-                Console.ReadLine();
+                }
+                else if (answer == "leave")
+                {
+                    Console.WriteLine(">You leave the firearm and rush to the door, hoping you don't need it");
+                }
+                
+
+
+                Console.WriteLine("");
+                Console.WriteLine("(Press any key to continue...)");
+                Console.ReadKey();
+
+                Console.WriteLine("You should be fine after all of this time.");
+                Console.WriteLine(">you stand up and use the bed as a crutch as you try to walk towards the door");
```

Comment:
Yes, that's all leftover plot.  I'm trying to decide if I should scrap everything there and below it or try to make it of use later in the ta.